Packs/fireeye/Integrations/integration-fireeye.yml

```diff
@@ -259,9 +259,10 @@ script:
            httpParams.Username = params.credentials.identifier;
            httpParams.Password = params.credentials.password;
         }
+        full_url = server + url + (method === 'GET' && method ? encodeToURLQuery(args) : '')
```

Comment:
why do we need the `&& method` part?

Reply:
Don't know just used the existing reference. Since we don't have an instance I don't want to change this and possibly break things.